### PR TITLE
Remove Desktop App

### PR DESCRIPTION
**Pull request by MTSaeed**

### CODE REVIEW

File: src/common/components/notification-handler/index.tsx [hunk 1]
```diff
@@ -37,7 +37,6 @@ class NotificationHandler extends Component<Props> {
 
     this.nws
       .withActiveUser(activeUser)
-      .withElectron(global.isElectron)
```
[the remark this answers is not in the document]
Comment (MTSaeed) [reply src/common/components/notification-handler/index.tsx:40]: Yes I have checked and learnt that the withElectron function sets the isElectron variable inside the  NotificationsWebSocket class. This variable is then used as an argument in the playNotificationSound. Now that we are moving away from electron I believe this function and the variable is no longer required. Therefore I have removed withElectron from here and also removed the definition.

**Comment by MTSaeed** [comment]
> Also we have to remove building in `gitlab_ci.yml`

I tried but I could not find this file. Can you please point out this file for me.

**Comment by MTSaeed** [comment]
@dkildar All comments have been resolved.

**Comment by feruzm** [comment]
> > Also we have to remove building in `gitlab_ci.yml`
> 
> I tried but I could not find this file. Can you please point out this file for me.

It is in .github folder there is desktop build ci files but no worries, we will remove that as we will remove desktop branch after this merge, that CI works only when `desktop` branch changes

File: src/common/components/landing-page/index.tsx [hunk 2]
```diff
@@ -319,55 +302,21 @@ export default (props: any) => {
...
-                onClick={() =>
-                  isElectron() &&
-                  window.open(
-                    "https://desktop.ecency.com/",
-                    "_blank",
-                    "top=500,left=200,frame=false,nodeIntegration=no"
-                  )
-                }
-              >
+              <Link to={"https://desktop.ecency.com/"} className="windows" target="blank">
```
Comment (feruzm) [inline src/common/components/landing-page/index.tsx:305]: No need to offer download desktop app links anymore. @MTSaeed @dkildar

**Review by feruzm — APPROVED** [review]
lgtm, only removing download links to desktop app should be done on same PR and also building project needs to be tested.

**Pull request merged**